xonsh/aliases.py

```diff
@@ -11,16 +11,20 @@ def cd(args, stdin=None):
     If no directory is specified (i.e. if `args` is None) then this
     changes to the current user's home directory.
     """
+    cur_oldpwd = builtins.__xonsh_env__['OLDPWD']
```

Comment:
I think that this would be a bug if `$OLDPWD` is not already set - which it isn't on my system :)  You probably use the `get()` method on the environment with some sensible default, like `os.getcwd()` or None.